src/swapelement.java

```diff
@@ -0,0 +1,22 @@
+import java.util.Arrays;
+
+public class swapelement {
+
+    public static void main(String[] args) {
+        int[] arr = {2, 9, 0, 44, 58, 289, 3, 90, 22, 234, 897};
+        int[] arr2 = new int[4];
```

Comment:
int[] arr2 = new int[4]; 
@Piyush-tft10 make int[4] as dynamic instead of static